core/cmd/presenters.go

```diff
@@ -135,6 +144,10 @@ func (j Job) FriendlyCreatedAt() string {
 		if j.OffChainReportingSpec != nil {
 			return j.OffChainReportingSpec.CreatedAt.Format(time.RFC3339)
 		}
+	case KeeperJob:
+		if j.KeeperSpec != nil {
+			return j.KeeperSpec.CreatedAt.Format(time.RFC3339)
```

Comment:
Why use this time format over ISO?

Reply:
I'm not sure. That was the one that was being used previously so I kept it the same